sphinx/ext/imgmath.py

```diff
@@ -90,15 +90,35 @@ class InvokeError(SphinxError):
 '''
 
 depth_re = re.compile(br'\[\d+ depth=(-?\d+)\]')
+depthsvg_re = re.compile(br'.*, depth=(.*)pt')
 
 
-def generate_latex_macro(math, config, confdir=''):
-    # type: (str, Config, str) -> str
+def read_svg_depth(filename):
+    # type: (str) -> int
+    """Read the depth from comment at last line of SVG file
+    """
+    with open(filename, 'r') as f:
+        for line in f:
+            pass
+        return int(line[11:-4])
```

Comment:
I feel this is a bit fragile. Could you replace this by regexp?

Reply:
I think you mean to use a regex for the parsing of very last line (cf `write_svg_depth()`), and we can pass over silently other lines?